code/src/sixsq/nuvla/server/resources/spec/infrastructure_service.cljc

```diff
@@ -90,6 +90,26 @@
              :json-schema/order 24
              :json-schema/hidden true)))
 
+
+(s/def ::swarm-enabled
```

Comment:
Since this is the generic spec for infrastructure service, can you put the `swarm-enabled` attribute as a tag? e.g. Since VPN infrastructure service should not have this attribute and make no sense to have it.

Reply:
I'd rather not because the only use we've had for tags so far is to label IS, in such a way that is the future we can use tags for placement policies. This isn't really a label that can be used for that, but rather a capability of the IS. Some IS will have it some others won't. 

if it can be moved to the swarm-template, that's ok, as long as it ends up in the final IS resource and can be edited